packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,48 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+import {randomBytes} from 'crypto';
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+if (!telClient.commonProperties.sessionId) {
+  telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
+  function sanitizeStackTrace(envelope: any /*context: any*/): boolean {
+    if (envelope.data.baseType === 'ExceptionData') {
+      const data = envelope.data.baseData;
+      if (data.exceptions && data.exceptions.length > 0) {
+        for (let i = 0; i < data.exceptions.length; i++) {
+          const exception = data.exceptions[i];
+          for (const frame of exception.parsedStack) {
+            const parens = frame.method.indexOf('(');
+            if (parens !== -1) {
+              // case 1: method === 'methodName (rootOfThePath'
+              frame.method = frame.method.substr(0, parens).trim();
```

Comment:
If we collect filenames fromt he callstack we should maek sure we santize the file paths as well since they might contain user names and/or company/domain names... if Users use the tools on their profile folders.

Reply:
This will not capture usernames or company or domain names, the only thing captured are function names, and the last part of the directory (the leaf directory) and filename. All these scripts will be under node_modules, several levels deep, so there is no risk that the username or other PII leaks onto the leaf directory.